crates/orchestrator/src/jobs/mod.rs

```diff
@@ -192,16 +229,43 @@ pub async fn process_job(id: Uuid, config: Arc<Config>) -> Result<(), JobError>
     .await
     .map_err(|e| JobError::Other(OtherError(e)))?;
 
+    tracing::info!("JOB PROCESSED: {:?} & {:?} & {:?}", job.job_type, job.internal_id, job.id);
+
+    let attributes = [
+        KeyValue::new("job_type", format!("{:?}", job.job_type)),
+        KeyValue::new("type", "process_job"),
+        KeyValue::new("job", format!("{:?}", job)),
+    ];
+
+    block_gauge.record(job.internal_id.parse::<f64>().unwrap(), &attributes);
+
     Ok(())
 }
 
 /// Verifies the job and updates the status of the job in the DB. If the verification fails, it
 /// retries processing the job if the max attempts have not been exceeded. If the max attempts have
 /// been exceeded, it marks the job as timed out. If the verification is still pending, it pushes the
 /// job back to the queue.
+#[tracing::instrument(
+    name = "general_verify_job",
+    skip(config),
+    fields(job, job_type, internal_id, verification_status)
+)]
 pub async fn verify_job(id: Uuid, config: Arc<Config>) -> Result<(), JobError> {
+    let block_gauge = telemetry::global_meter()
+        .f64_gauge(format!("{:?}{}", OTEL_SERVICE_NAME, "_block_state"))
+        .with_description("A gauge to show block state at given time")
+        .with_unit("block")
+        .init();
+
     let mut job = get_job(id, config.clone()).await?;
 
+    tracing::Span::current().record("job", format!("{:?}", job.clone()));
```

Comment:
same as above

Reply:
**Discussed offline.**
Keeping as it is, since this is minimal changes.